### DISCUSSION TITLE
Possibility to start the circulation pump for warm water for e.g. 5 min?

### DISCUSSION THREAD

**Discussion by freakyjoe8** [fredlcore/BSB-LAN#549] [Q&A] [answered]
Hi everyone,

is it possible via BSB LAN to let the circulation pump run for a certain amount of time? I'd like to trigger the pump e.g. via a switch in case I need warm water and otherwise keep the circulation off.

Thanks!
Br

**Comment by fredlcore** [comment, marked as answer]
Such time-based functions could potentially be implemented using BSB_LAN_custom.h, but I'm not sure whether you can directly start and stop the circulation pump via BSB-LAN. You would have to see if there is a parameter on the heater that allows for that, and if there is, you can do it via BSB-LAN, but if there isn't, then there's nothing you can do about it.
